enhancements/oc/cli-manager.md

```diff
@@ -0,0 +1,273 @@
+---
+title: cli-manager
+authors:
+  - "@sallyom"
+  - "@deejross"
+reviewers:
+  - "@soltysh"
+  - "@jwmatthews"
+approvers:
+  - "@soltysh"
+  - "@sferich888"
+  - "@deads2k"
+  - "@spadgett"
+creation-date: 2021-10-06
+last-updated: 2021-10-18
+status: implementable
+---
+
+# OpenShift CLI Manager
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
```

Comment:
The above two can be checked, given we're planning to start implementation in the next release.